x/stakeibc/handler.go

```diff
@@ -67,6 +67,9 @@ func NewMessageHandler(k keeper.Keeper) sdk.Handler {
 		case *types.MsgUpdateInnerRedemptionRateBounds:
 			res, err := msgServer.UpdateInnerRedemptionRateBounds(sdk.WrapSDKContext(ctx), msg)
 			return sdk.WrapServiceResult(ctx, res, err)
+		case *types.MsgResumeHostZone:
```

Comment:
Should we test this on dockernet as well?

Reply:
Definitely